extensions/typescript-language-features/src/languageFeatures/quickFix.ts

```diff
@@ -313,22 +353,27 @@ class TypeScriptQuickFixProvider implements vscode.CodeActionProvider<VsCodeCode
 		diagnostic: vscode.Diagnostic,
 		tsAction: Proto.CodeFixAction
 	): CodeActionSet {
-		results.addAction(this.getSingleFixForTsCodeAction(document.uri, diagnostic, tsAction));
+		results.addAction(this.getSingleFixForTsCodeAction(document, diagnostic, tsAction));
 		this.addFixAllForTsCodeAction(results, document.uri, file, diagnostic, tsAction as Proto.CodeFixAction);
 		return results;
 	}
 
 	private getSingleFixForTsCodeAction(
-		resource: vscode.Uri,
+		document: vscode.TextDocument,
 		diagnostic: vscode.Diagnostic,
 		tsAction: Proto.CodeFixAction
 	): VsCodeCodeAction {
+		const aiQuickFixEnabled = vscode.workspace.getConfiguration('typescript').get('experimental.aiQuickFix');
+		let followupAction: Command | undefined;
+		if (aiQuickFixEnabled && tsAction.fixName === fixNames.classIncorrectlyImplementsInterface) {
+			followupAction = new EditorChatFollowUp('Implement the class using the interface', document, diagnostic.range, this.client);
```

Comment:
Should this string be localized?

Reply:
I honestly don't know. This string is being used a prompt for the LLM